### DISCUSSION TITLE
Customize creation process when using wizard

### DISCUSSION THREAD

**Discussion by crisjohn02** [filamentphp/filament#5787] [Help]
Hi, is it possible to customized creation process when using wizard? I've been trying to load the users relationship thru wizard and I want to create a team after creating the related user.

Thank you..

**Comment by danharrin** [comment]
It's no different to normal, you probably want the afterCreate() hook

**Reply by crisjohn02** [reply]
hi, good day.

Sorry but I think the CreateAction::make() only works in modals and I was using the wizard.

I also tried this one https://filamentphp.com/docs/2.x/admin/resources/relation-managers#customizing-the-creation-process but it only works in modal too.

If I use the afterCreate() hook, how would I get the added related row? thank you so much

**Reply by danharrin** [reply]
In `afterCreate()`, the newly created record is `$this->record`
